flight/states/takeoff.py

```diff
@@ -29,4 +31,18 @@ async def run(self, drone: System) -> Waypoints:
         Waypoints : State
             Next state to fly waypoint path
         """
+        # Initialize altitude (convert feet to meters)
+        await drone.action.set_takeoff_altitude(75 / 3.2808)
```

Comment:
Here I would import the constant TAKEOFF_ALT from the config.py file here, so it doesn't seem like this value is appearing out of thin air.